articles/quickstart/spa/auth0-react/01.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to a React application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - react
+  - login
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+This guide demonstrates how to use the Auth0 React SDK with any new or existing React application.
+
+## Prerequisites
+
+- If you don't have an Auth0 account already, <a href="https://auth0.com/signup" data-amp-replace="CLIENT_ID" data-amp-addparams="anonId=CLIENT_ID(cid-scope-cookie-fallback-name)">sign up for a free Auth0 account</a>, and [create a tenant](https://auth0.com/docs/getting-started/create-tenant).
+
+- [Register your React application](https://auth0.com/docs/dashboard/guides/applications/register-app-spa) as a single-page application with Auth0.
+
+## Install the SDK
+
+Run the following command within your project directory to install the Auth0 React SDK:
+
+```bash
+npm install @auth0/auth0-react
+```
+
+> You may use `yarn` instead.
+
+You can use the Auth0 React SDK to add authentication to your React app quickly, gain access to user profile information, and securely call APIs.
+
+## Configure the SDK
+
+Under the hood, the Auth0 React SDK uses [React Context](https://reactjs.org/docs/context.html) to manage the authentication state of your users and to expose helpful utility methods. One way to integrate Auth0 with your React app is to wrap your root component with an `Auth0Provider` that you can import from the SDK.
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+import {Auth0Provider} from "@auth0/auth0-react";
+
+ReactDOM.render(
+  <Auth0Provider 
+    domain="${account.namespace}"
+    client_id="${account.clientId}"
+    redirect_uri={window.location.origin}
+  >
+    <App />
+  </Auth0Provider>,
+  document.getElementById("root")
+);
+```
+
+The `Auth0Provider` component takes three props:
+ 
+- `domain` and `client_id`: The values of these properties correspond to the "Domain" and "Client ID" values present under the "Settings" of the single-page application that you registered with Auth0.
```

Comment:
Please also explain how to use your custom domain if configured. This will not show up in the "Domain" field.

Reply:
Sandrino, I don't think this is something that has been covered in a quickstart before. Perhaps, is it something that rather belongs to the docs? We've discussing within my team as well what should be the topic coverage and depth of a quickstart as their maintenance and creation will be passed to us.

Reply:
We need to add a link to docs then. Copying the "Domain" from the settings page is not something you do when using a custom domain.